lib/Doctrine/ORM/Repository/InvalidMagicMethodCall.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Repository;
+
+use Doctrine\ORM\RepositoryException;
+
+final class InvalidMagicMethodCall extends \InvalidArgumentException implements RepositoryException
+{
+    public static function fromEntityNameFieldNameAndMethod(
```

Comment:
This method name is a bit long, I'm open to proposals

Reply:
`fieldNameNotFound` or `fieldNotFound` because it's the reason for not be able to use a magic method in an entity.

Reply:
So no `from`?

Reply:
The static name may lack fluent reading because of a missing method prefix like "from". Sounds funny with "because" `InvalidMagicMethodCall::becauseFieldNotFoundIn`, but I like "In" as suffix. Maybe the suffix can make it more readable without having needed arguments in the method name.